docs/pt/docs/advanced/response-directly.md

```diff
@@ -0,0 +1,70 @@
+# Retornando uma Resposta Diretamente
+
+Quando você cria uma *operação de rota* no **FastAPI** você pode retornar qualquer dado nela: um dicionário (`dict`), uma lista (`list`), um modelo do Pydantic ou banco de dados, etc.
```

Comment:
It got a little weird. It looks like you can return the database

"*um modelo do Pydantic ou banco de dados*"

I would add a "*ou do do banco*"